scripts/gateway.service

```diff
@@ -0,0 +1,39 @@
+# This is a SystemD unit file for the Gateway
+# To configure:
+# - Update the user and group that the service will run as (User & Group below) or create one with:
+#   'sudo adduser --system --no-create-home --group --disabled-login storj'
+# - Ensure that the gateway binary is in /usr/bin and is named gateway (or edit the ExecStart line
```

Comment:
Shouldn't we also comment that the file has the permission to be executed by others or change the owning group to `storj` and/or owner to `storj`?

Reply:
Executables in /usr/bin are expected to be accessible by all users on the system. I would strictly avoid changing the permissions of those files in that path.

Reply:
I think the same, but I gave 3 options as an alternative, the first one was if we have to indicate to make sure that the file has the executable permission set to others.

Reply:
I do not think that we have to specify that here. Someone that uses systemd services should be aware of that.

Reply:
yeah, things in /usr/bin (or /usr/local/bin) should definitely be owned by root. otherwise, it's a security risk for other users on the system if the owning user is compromised and changes the binary in a PATH location.

i would also suggest we recommend /usr/local/bin instead of /usr/bin, to comply better with FHS (https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard)

Reply:
Agreed, that what i had initially.

Reply:
> I do not think that we have to specify that here. Someone that uses systemd services should be aware of that.

Probably, but also that person would likely aware of setting the right permissions to configuration file, so we are already pedantic on that so I don't see that it hurts to be pedantic on it too.